File: src/prob9.py
count = 100
triplets = []
a = 0
b = 0
c = 0
sumOfNums = 0
product = 1
while sumOfNums != 1000:
    count += 1
    for k in range(1,count + 1):
        a = k
        for i in range(1,count + 1):
            b = i
            for j in range(1,count + 1):
                c = j
                if (a*a) + (b*b) == (c*c) and a < b and b < c:
                    if (a,b,c) in triplets:
                        continue
                    triplets.append((a,b,c))
                    print("Pythagorean Triplet found, testing product")
                    print("a: " + str(a) + "\nb: " + str(b) + "\nc: " + str(c))                     
                    #print(triplets)
                    product = a * b * c
                    sumOfNums = a + b + c 
                    print("Sum Of Nums : " + str(sumOfNums)) 
    
print("finished!")
print("a: " + str(a) + "\nb: " + str(b) + "\nc: " + str(c))
print(product)
print(triplets)

File: src/java/prob9.java
public class prob9 {
    public static void main(String[] args){
        int product = 0;
        for (int i = 0; i < 1000; i++) {
            for (int j = 0; j < 1000; j++) {
                for (int k = 0; k < 1000; k++) {
                    if ((i * i) + (j * j) == (k * k) && i < j && i < k && j < k) {
                        if ((i + j + k) == 1000) {
                            product = i * j * k;
                            System.out.println("Pythagorean triplet found : " + i + "," + j + "," + k);
                            System.out.println("Solution: " + product);
                        }
                    }
                }
            }
        }
    }
}
